src/swap/components/SwapProvider.tsx

```diff
@@ -42,6 +42,7 @@ export function SwapProvider({
   children: React.ReactNode;
   experimental: {
     useAggregator: boolean; // Whether to use a DEX aggregator. (default: true)
+    slippage?: string; // Maximum acceptable slippage for a swap. (default: 10) This is as a percent, not basis points
```

Comment:
Based on your comment, should this be called `maxSlippage` and being a `number` instead of a string?